crates/oidc-client/src/requests/userinfo.rs

```diff
@@ -98,16 +99,17 @@ pub async fn fetch_userinfo(
         .call(userinfo_request)
         .await?;
 
-    let content_type = userinfo_response
+    let content_type: Mime = userinfo_response
         .headers()
         .get(CONTENT_TYPE)
```

Comment:
Maybe we should use [`.typed_try_get()`](https://docs.rs/headers/latest/headers/trait.HeaderMapExt.html#tymethod.typed_try_get) here instead?

Reply:
`mas-oidc-client` already depends on `headers`, so yeah let's use this instead 👍

Reply:
Done.